test/subscriptions/subscriptionRequest.json

```diff
@@ -16,5 +16,6 @@
         "warningLevel"
       ]
     }
-  ]
+  ],
+  "attributes":["sleepTime","warningLevel"]
```

Comment:
Why this change was needed? Were test not passing without the change?

Reply:
The update of Nock module implies that payloads must have exact matching. This was also reported in a previous PR of iotagent-node-lib: https://github.com/telefonicaid/iotagent-node-lib/issues/660#issuecomment-421041564
So in order to pass this test, we needed to complete the nock payload.